From 7795997c9454783d73c02f1e7c253f14a268229e Mon Sep 17 00:00:00 2001
From: Ilies CHERGUI <ilies.chergui@gmail.com>
Date: Sun, 17 Apr 2022 00:45:27 +0100
Subject: [PATCH 2/5] mk/common.mk: set JETSON variable if not set before

changing the equal sign will allow modifying the variable using the
environment variables.

Signed-off-by: Pablo Rodriguez Quesada <pablo.rodriguez-quesada@windriver.com>
Signed-off-by: Ilies CHERGUI <ilies.chergui@gmail.com>
---
 mk/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mk/common.mk b/mk/common.mk
index d6ef499..875e412 100644
--- a/mk/common.mk
+++ b/mk/common.mk
@@ -24,7 +24,7 @@ DATE     := $(shell date -u --iso-8601=minutes)
 REVISION := $(shell git rev-parse HEAD)
 COMPILER := $(realpath $(shell which $(CC)))
 PLATFORM ?= $(shell uname -m)
-JETSON   := TRUE
+JETSON   ?= TRUE
 
 ifeq ($(DATE),)
 $(error Invalid date format)
-- 
2.32.0

